doc-content/enterprise-only/openshift/imagestreams-file-install-proc.adoc

```diff
@@ -84,6 +84,31 @@ $ oc create -f {PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-image-streams.yaml -n op
 endif::[]
 +
 *** Using the OpenShift Web UI, select *Add to Project -> Import YAML / JSON* and then choose the file or paste its contents.
+ifeval::["{context}"=="openshift-authoring"]
++
+. If you want to deploy a high-availability {CENTRAL} (this functionality is tech preview), complete the following additional steps:
```

Comment:
Minor suggestion - maybe changing this to a note to highlight that this is technology preview:

```
Note
Configuration of {CENTRAL} for high availability is currently technology preview.
```